src/dbt_score/cli.py

```diff
@@ -48,9 +53,10 @@ def cli() -> None:
     "--select",
     "-s",
     help="Specify the nodes to include.",
-    cls=MultiOption,
+    cls=MultiOption if DBT_INSTALLED else None,
```

Comment:
Makes me wonder if we should remove that feature, and enforce `--select foo --select bar` instead of `--select foo bar`.

Reply:
To be honest I like how we have the exact same select statement that `dbt` uses, it will prevent any confusion on how users are able to select models (and sources) in the future